docs/security.md

```diff
@@ -3,96 +3,12 @@ post_title: Security
 menu_order: 40
 enterprise: 'no'
 ---
-# Mesos Security
 
-## SSL
-
-<table class="table">
```

Comment:
Why remove the contents of this file?  Where is the source for https://docs.mesosphere.com/service-docs/spark/spark-auth/?

I wrote this file because I think Spark-specificy security docs should live with the Spark project.